doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -458,6 +458,32 @@ Field        | Type                 | Meaning
 `owner_id`   | [`text`]             | The role ID of the owner of the table. Corresponds to [`mz_roles.id`](/sql/system-catalog/mz_catalog/#mz_roles).
 `privileges` | [`mz_aclitem array`] | The privileges belonging to the table.
 
+### `mz_timezone_abbrevs`
+
+The `mz_timezone_abbrevs` view contains a row for each supported timezone abbreviation.
+A "fixed" abbreviation does not change its offset or daylight status based on the current time.
+A non-"fixed" abbreviation is dependent on the current time for its offset, and must use the [`timezone_offset`](/sql/functions/#timezone_offset) function to find its properties.
+These correspond to the `pg_catalog.pg_timezone_abbrevs` table, but can be materialized as they do not depend on the current time.
+
+<!-- RELATION_SPEC mz_catalog.mz_timezone_abbrevs -->
+Field        | Type                 | Meaning
+-------------|----------------------|----------
+`abbrev`     | [`text`]             | The timezone abbreviation.
```

Comment:
Should we designate `abbrev` as one of the allowed abbreviations? https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#system-catalog-style

I think it could make sense, given that PostgreSQL uses it in `pg_timezone_abbrevs` already.

Reply:
No, I changed all these that mz owns to "abbreviation[s]".